easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch

```diff
@@ -0,0 +1,13 @@
+# Don't use libmpc and libmpfr to find libisl
```

Comment:
@wpoely86 please add yourself as an author

does it make sense to contribute this upstream?